documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,401 @@
+            { type: "html", subdir: "html" },
+            { type: "cobertura", subdir: ".", file: "cobertura-coverage.xml" }
+          ]
+        },
+    
+        junitReporter: {
+          outputDir: "", // results will be saved as $outputDir/$browserName.xml
+          outputFile: "test-results.browser.xml", // if included, results will be saved as $outputDir/$browserName/$outputFile
+          suite: "", // suite will become the package name attribute in xml testsuite element
+          useBrowserName: false, // add browser name to report and classes names
+          nameFormatter: undefined, // function (browser, result) to customize the name attribute in xml testcase element
+          classNameFormatter: undefined, // function (browser, result) to customize the classname attribute in xml testcase element
+          properties: {} // key value pair of properties to add to the <properties> section of the report
+        },
+    
+        jsonToFileReporter: {
+          filter: jsonRecordingFilterFunction,
+          outputPath: "."
+        },
+    
+        // web server port
+        port: 9876,
+    
+        // enable / disable colors in the output (reporters and logs)
+        colors: true,
+    
+        // level of logging
+        // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+        logLevel: config.LOG_INFO,
+    
+        // enable / disable watching file and executing tests whenever any file changes
+        autoWatch: false,
+    
+        // --no-sandbox allows our tests to run in Linux without having to change the system.
+        // --disable-web-security allows us to authenticate from the browser without having to write tests using interactive auth, which would be far more complex.
+        browsers: ["ChromeHeadlessNoSandbox"],
+        customLaunchers: {
+          ChromeHeadlessNoSandbox: {
+            base: "ChromeHeadless",
+            flags: ["--no-sandbox", "--disable-web-security"]
+          }
+        },
+    
+        // Continuous Integration mode
+        // if true, Karma captures browsers, runs the tests and exits
+        singleRun: false,
+    
+        // Concurrency level
+        // how many browser should be started simultaneous
+        concurrency: 1,
+    
+        browserNoActivityTimeout: 60000000,
+        browserDisconnectTimeout: 10000,
+        browserDisconnectTolerance: 3,
+        browserConsoleLogOptions: {
+          terminal: !isRecordMode()
+        },
+    
+        client: {
+          mocha: {
+            // change Karma's debug.html to the mocha web reporter
+            reporter: "html",
+            timeout: "600000"
+          }
+        }
+      });
+    };
+    ```
+1. **add test utils.**  
+
+    create a test/public folder and then copy the content [here](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/test/public/utils) into public folder
```

Comment:
I think we need a bit more details here. I think you should explain that here we create the client and plug it into the recorder so that we can test without calling the actual service.

Also we need to note here that there are some manual changes in the copied files that need to be done. For example

```typescript
import PurviewAccount, { PurviewAccountRestClient } from "../../../src";
```

Needs to change to the value that was used in the swagger/readme.md in `title`. For example if title is "Foo"
```typescript
import Foo, { FooRestClient } from "../../../src";
```

and 

```typescript
export function createClient(options?: ClientOptions): PurviewAccountRestClient {
  const credential = new ClientSecretCredential(
    env.AZURE_TENANT_ID,
    env.AZURE_CLIENT_ID,
    env.AZURE_CLIENT_SECRET
  );
  return PurviewAccount(env.ENDPOINT, credential, options);
}
```

Needs to change to

```typescript
export function createClient(options?: ClientOptions): FooRestClient {
  const credential = new ClientSecretCredential(
    env.AZURE_TENANT_ID,
    env.AZURE_CLIENT_ID,
    env.AZURE_CLIENT_SECRET
  );
  return Foo(env.ENDPOINT, credential, options);
}
```

Reply:
updated.